src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -870,6 +870,8 @@ where constraint_type = 'PRIMARY KEY' and tc.table_catalog = '${this.dbName}'`;
                 c += " DEFAULT " + column.default + "";
             }
         }
+        if (column.isGenerated && column.type === "uuid")
+            c += " DEFAULT uuid_generate_v4()";
```

Comment:
There are several other uuid functions, but everyone other ORM I have used uses the v4 function.  https://www.postgresql.org/docs/9.5/static/uuid-ossp.html... Fixing the other issue you brought up, still allowing for a default value to be set when generated, should give users the ability to use another uuid function.